tools/test.py

```diff
@@ -34,25 +19,14 @@ def main():
 
     enable_ansi_colors()
 
-    test_cases = [
-        TestSetup,
-        TestUtil,
-        TestTarget,
-        JsUnitTests,
-        TestFetch,
-        TestRepl,
-        TestDenoDir,
-        TestBenchmark,
-        TestIsTty,
-    ]
-    test_cases += permission_prompt_tests()
-    test_cases += complex_permissions_tests()
-    # It is very slow, so do TestFmt at the end.
-    test_cases += [TestFmt]
-
-    with http_server.spawn():
-        run_tests(test_cases)
+    cargo_test = ["cargo", "test", "--locked"]
+    if "DENO_BUILD_MODE" in os.environ and \
+      os.environ["DENO_BUILD_MODE"] == "release":
+        run(cargo_test + ["--release"])
+    else:
+        run(cargo_test)
 
 
 if __name__ == '__main__':
-    main()
+    with http_server.spawn():
```

Comment:
I imagine you're keeping this script because of need to spawn http server. If we're migrating all tests to Rust I believe it makes sense to use some kind of Rust testing framework that allows to spawn additional tasks before running tests.

Reply:
Yes, this script remains only for spawning http_server before tests and stopping it after tests. As you mentioned in twitter, rust doesn't seem providing an easy simple way for doing that. So I kept this script for now.